test/suites/examples/install-karpenter.yaml

```diff
@@ -0,0 +1,115 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: install-karpenter-task
+  namespace: tekton
+spec:
+  description: |
+    Install Karpenter and necessary resources onto an EKS cluster.
+  params:
+  - name: cluster-name
+    description: The name of the cluster to install Karpenter
+  - name: karpenter-version
+    description: The version of Karpenter to install. Needs to be in vx.y.z format where x, y, and z are integers.
+  - name: account-id
+    description: Account ID where to deploy resources
+  - name: pre-install-image
+    description: Docker image for the Tekton pod to install resources. This needs minimally needs awscli, kubectl, and curl.
+  - name: aws-region
+    description: AWS Region where cluster is deployed
+  workspaces:
+  - name: config
+    description: |
+      A workspace into which a kubeconfig file called `kubeconfig` will be written that will contain the information required to access the cluster.
+  steps:
+  - name: karpenter-pre-install
+    image: $(params.pre-install-image)
+    script: |
+        TEMPOUT=$(mktemp)
```

Comment:
should we add shebang here? 

```
#!/usr/bin/env bash
set -euo pipefail
```

Reply:
No shebangs are needed, it automatically assumes it's bash, and also fails if any command fails too.